modules/openxBidAdapter.js

```diff
@@ -26,35 +27,66 @@ export const spec = {
       return;
     }
 
-    let delDomain = bids[0].params.delDomain;
-    let configuredBc = bids[0].params.bc;
-    let bc = configuredBc || `${BIDDER_CONFIG}_${BIDDER_VERSION}`;
-
-    return buildOXRequest(bids, {
-      ju: currentURL,
-      jr: currentURL,
-      ch: document.charSet || document.characterSet,
-      res: `${screen.width}x${screen.height}x${screen.colorDepth}`,
-      ifr: isIfr,
-      tz: new Date().getTimezoneOffset(),
-      tws: getViewportDimensions(isIfr),
-      ef: 'bt%2Cdb',
-      be: 1,
-      bc: bc,
-      nocache: new Date().getTime()
-    },
-    delDomain);
+    let requests = [];
+    let banner_requests = [];
+    let video_requests = [];
+    let bids_banner = bids.filter(function(bid) { return bid.mediaType !== 'video'; });
```

Comment:
Are there other types than just banner and video?  If so, switch to a strict check on the type

Reply:
cant we use the const in src/mediaTypes?